config.json

```diff
@@ -1791,6 +1791,17 @@
           "lists"
         ],
         "difficulty": 10
+      },
+      {
+        "slug": "game-of-life",
+        "name": "Conway's Game of Life",
+        "uuid": "a5f8aef3-9661-49c7-9eb3",
+        "practices": ["multi-dimensional-arrays"],
```

Comment:
Please leave `practices` empty as it's going to get treated in the following [issue](https://github.com/exercism/java/issues/2631)